File: Aplitest/app/src/main/java/com/example/aplitest/WeatherService.java
package com.example.aplitest;

import retrofit2.Call
import retrofit2.http.GET
interface WeatherService {

    @GET(value: "?q=Paris&appid=ee2b8d7cd97c82ff15d117e5556e5eb3&units=metric")
    fun getWeatherByCity(): Call<String>
}

File: Aplitest/app/src/main/java/com/example/aplitest/WeatherService.kt
package com.example.aplitest

import com.google.gson.JsonObject
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query

interface WeatherService {
    companion object {
        const val API_KEY = "ee2b8d7cd97c82ff15d117e5556e5eb3"
    }
    @GET("?&appid=$API_KEY&units=metric")
    fun getWeatherByCity(@Query("q") city: String): Call<weatherResult>
}
